test/mock/mock-arc

```diff
@@ -58,6 +58,10 @@ messages
   msgID *String
   posted **String
 
+accounts-messages
+  accountID *String
+  msgID *String
```

Comment:
Oh man took my while, as I'm adding more integration tests, to catch that two hash keys were provided here 🙈